src/translations/screens/Onboarding.ts

```diff
@@ -21,6 +21,22 @@ const OnboardingTexts = {
     ),
     mainButton: _('Neste', 'Next'),
   },
+  goodToKnow: {
+    title: _('Greit å vite', 'Good to know'),
+    description: _(
+      'Når du gjør et reisesøk får du opp forslag til hvordan du kan reise, og en korrespondanse som er garantert er merket med korrespondanse.',
```

Comment:
```suggestion
      'Når du gjør et reisesøk får du opp forslag til hvordan du kan reise, og en korrespondanse som garantert er merket med korrespondanse.',
```

Reply:
I think we should leave this as is or ask Christoffer for a new text if it is unclear. They want to state clearly that only guaranteed correspondences are marked with correspondence

Reply:
Okay. Think I understood the sentence now after reading it six times. Suggested reword by AI: "Når du søker etter reisemuligheter, vil du få forslag til ulike reisealternativer, og korrespondansen som er bekreftet vil være tydelig merket."